tests/codegen/i128-x86-callconv.rs

```diff
@@ -0,0 +1,76 @@
+//! Verify that Rust implements the expected calling convention for `i128`/`u128`.
+
+//@ revisions: MSVC MINGW
+//@ add-core-stubs
+//@ [MSVC] needs-llvm-components: x86
+//@ [MINGW] needs-llvm-components: x86
+//@ [MSVC] compile-flags: --target x86_64-pc-windows-msvc
+//@ [MINGW] compile-flags: --target x86_64-pc-windows-gnu
+//@ [MSVC] filecheck-flags: --check-prefix=WIN
+//@ [MINGW] filecheck-flags: --check-prefix=WIN
```

Comment:
Suggestion (blocked on a compiletest change):

```
//@ revisions: MSVC MINGW WIN
//@[MSVC] only-msvc
//@[MINGW] only-windows-gnu
//@[WIN] only-windows
```

Reply:
Wouldn't this mean that I can't run the tests on anything other than Windows?

Reply:
You're right, ignore my previous comment :P

Reply:
The `minicore` experience is wonderful :) I just used it here since I was developing on a non-Windows machine, but it might be convenient to nudge more codegen tests toward using `minicore` rather than relying on native platforms and a lot of debug-by-CI.

Or maybe there could be a flag to use `minicore` on non-native targets but the real `core` otherwise. Hm....

Reply:
> Or maybe there could be a flag to use `minicore` on non-native targets but the real `core` otherwise. Hm....

If you use the real core, then you'd need to build the full libcore for native `$HOST`, right?

Reply:
~~I was assuming that~~ actually I'll move this to Zulip rather than finishing typing it here since there is probably a bit of digging that we could do here